x/stakeibc/module_simulation.go

```diff
@@ -24,7 +25,7 @@ var (
 )
 
 const (
-	opWeightMsgRestoreInterchainAccount = "op_weight_msg_register_interchain_account"
+	opWeightMsgRestoreInterchainAccount = "op_weight_msg_register_interchain_account" // #nosec
```

Comment:
why nosec here?

Reply:
it was flagging it as a private key for some reason